admin/spec/service/jwt.service.spec.js

```diff
@@ -139,10 +138,10 @@ describe('JWT service', () => {
       })
     })
 
-    describe('and the pupil has had the key revoked', () => {
+    fdescribe('and the pupil has had the key revoked', () => {
```

Comment:
`fdescribe`: is this debug?

Reply:
@jon-shipley removed since the tests are passing now